src/neural/dx/MetaCommand.h

```diff
@@ -0,0 +1,128 @@
+/*
+  This file is part of Leela Chess Zero.
+  Copyright (C) 2020 The LCZero Authors
+
+  Leela Chess is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  Leela Chess is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with Leela Chess.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
```

Comment:
We also insert additional permission in our files:

```
  Additional permission under GNU GPL version 3 section 7

  If you modify this Program, or any covered work, by linking or
  combining it with NVIDIA Corporation's libraries from the NVIDIA CUDA
  Toolkit and the NVIDIA CUDA Deep Neural Network library (or a
  modified version of those libraries), containing parts covered by the
  terms of the respective license agreement, the licensors of this
  Program grant you additional permission to convey the resulting work.
```

Reply:
I understand the additional permission is not needed in a backend, as long as we are not distributing it with the cuda/cudnn dlls. However, I hope @ankan-ban is OK with adding it later if we need it.

Reply:
I am fine with either option - whatever makes it easier to distribute lc0. 
The dx backend files don't use any of cuda/cudnn libraries so I left that part out (similar to OpenCL backend's copyright string).

Reply:
Adding the permission makes it easier for possible future decisions.
E.g. why not include this backend in all builds?

Reply:
ok. I have added that to all the files in dx backend.